### DISCUSSION TITLE
Add Reverse Proxy For NPMPlus

### DISCUSSION THREAD

**Discussion by Jecht88** [ZoeyVid/NPMplus#607] [Q&A] [answered]
Hello, I am trying to reverse proxy NPMPlus itself. When I do I get "npm.domain.com:81", and seems to break NPMPlus inside Unraid. I end up having to delete my config file. Not sure what im doing wrong, and why its adding the port number at the end. Im fine with other reverse proxy's "overseerr.domain.com" will setup just fine. Any help would be appreciated, Im at the end of my knowledge base.

**Comment by Zoey2936** [comment, marked as answer]
please send screenshots of your config
